views/index.handlebars

```diff
@@ -1,4 +1,4 @@
-<div id="page-one">
+<div id="page-one" hidden>
```

Comment:
Hides the content by default on the homepage, and then we show the content on `document.ready` using jQuery in frontend/src/upload.js above (assuming the user has JavaScript enabled).